### PR TITLE
Release 3.0 | Update TAN FAQs

### PR DESCRIPTION
**Pull request by MohamedZaghdoudi**
The PR is about the updates  of the Release 3.0 in FAQ-EN and FAQ-DE 

The date at the top is a placeholder. This is so that we can replace the dates of all affected FAQs at once when they're in the 3.0 branch.

image

image

---
Internal Tracking ID: [EXPOSUREAPP-14496](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14496)

### CODE REVIEW

**Comment by MikeMcC399** [comment]
@MohamedZaghdoudi 

The files README, package.json & package-lock.json should not be in this PR. You probably need to rebase onto the branch which you are merging into i.e. https://github.com/corona-warn-app/cwa-website/tree/release/3.0.

**Comment by MikeMcC399** [comment]
@MohamedZaghdoudi 

You can check your branch before you push it to GitHub by using:

`npm run test:short`

This will give the same error about the missing FAQ article `#warn_without_tan`.

The app should be called "Corona-Warn-App" not "Corona Warning App" as this is the name used everywhere else, including in the English app stores.

`D.h.` like all similar German abbreviations should be written with `&nbsp;` i.e.`D.&nbsp;h.`

I will look in more detail when the PR goes from Draft into review mode.

File: src/data/faq_de.json [hunk 1]
```diff
@@ -1434,6 +1447,8 @@
                                 "anchor": "tan_procedure",
                                 "active": false,
                                 "textblock": [
+                                    "<b>Aktualisiert am ?????r</b>",
+                                    "Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ?????h eingestellt.",
```
Comment (larswmh) [inline src/data/faq_de.json:1451]: Specified placeholder was ??????h, one ? is missing
```suggestion
                                    "Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ??????h eingestellt.",
```

File: src/data/faq_de.json [hunk 2]
```diff
@@ -1495,21 +1510,24 @@
...
-                                    "Die TAN-Hotline (<a href='#international_phone_numbers' target='_blank'>klicken Sie hier um zu den Telefonnummern zu gelangen</a>) hilft Ihnen bei der Eingabe der TAN, wenn Sie ein positives PCR-Testergebnis haben, und wenn Sie keine TAN oder kein Dokument mit PCR-Test-QR-Code erhalten haben. Sie erreichen sie von Montag bis Sonntag von 7 bis 20 Uhr Ortszeit in Deutschland. Der Anruf ist innerhalb Deutschlands kostenfrei.",
+                                    "Die TAN-Hotline (<a href='#international_phone_numbers' target='_blank'>klicken Sie hier um zu den Telefonnummern zu gelangen</a>) hilft Ihnen bei der Eingabe der TAN, wenn Sie ein positives PCR-Testergebnis haben, und wenn Sie keine TAN oder kein Dokument mit PCR-Test-QR-Code erhalten haben. Sie erreichen sie von Montag bis Sonntag von 7 bis 20 Uhr Ortszeit in Deutschland. Der Anruf ist innerhalb Deutschlands kostenfrei. Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ?????h eingestellt.",
```
Comment (larswmh) [inline src/data/faq_de.json:1515]: Specified placeholder was ??????h, one ? is missing
```suggestion
                                    "Die TAN-Hotline (<a href='#international_phone_numbers' target='_blank'>klicken Sie hier um zu den Telefonnummern zu gelangen</a>) hilft Ihnen bei der Eingabe der TAN, wenn Sie ein positives PCR-Testergebnis haben, und wenn Sie keine TAN oder kein Dokument mit PCR-Test-QR-Code erhalten haben. Sie erreichen sie von Montag bis Sonntag von 7 bis 20 Uhr Ortszeit in Deutschland. Der Anruf ist innerhalb Deutschlands kostenfrei. Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ??????h eingestellt.",
```

File: src/data/faq_de.json [hunk 3]
```diff
@@ -1518,7 +1536,9 @@
                                     "<b>TAN-Hotline</b>",
                                     "Die TAN-Hotline hilft Ihnen bei der Eingabe der TAN, wenn Sie ein positives PCR-Testergebnis haben, und wenn Sie keine TAN oder kein Dokument mit PCR-Test-QR-Code erhalten haben. Sie erreichen sie von Montag bis Sonntag von 7 bis 20 Uhr Ortszeit in Deutschland:",
                                     "<b>National:</b> 0800 7540002",
-                                    "<b>International:</b> +49 30 498 75402"
+                                    "<b>International:</b> +49 30 498 75402",
+                                    "Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ?????h eingestellt.",
```
Comment (larswmh) [inline src/data/faq_de.json:1540]: Specified placeholder was ??????h, one ? is missing
```suggestion
                                    "Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ??????h eingestellt.",
```

File: src/data/faq_de.json [hunk 4]
```diff
@@ -1581,17 +1601,19 @@
...
+                                    "Rufen Sie bitte beim zuständigen ärztlichen Fachpersonal bzw. Testcenter/Gesundheitsamt an, um Ihr Testergebnis zu erhalten. Wird Ihnen dabei ein positives Testergebnis mitgeteilt, können Sie über die TAN-Hotline eine teleTAN beantragen (<a href='#international_phone_numbers' target='_blank' >klicken Sie hier um zu den Telefonnummern zu gelangen</a>), um über die Corona-Warn-App Ihre Kontakte zu warnen. Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ?????h eingestellt. Bevor Sie die teleTAN in die App (Version 2.28 oder kleiner) eintragen können, müssen Sie erst das noch ausstehende Testergebnis aus der App löschen. Dazu tippen Sie bitte auf ",
```
Comment (larswmh) [inline src/data/faq_de.json:1610]: Specified placeholder was ??????h, one ? is missing

Version 2.28 oder kleiner -> Version 2.28 oder darunter 
```suggestion
                                    "Rufen Sie bitte beim zuständigen ärztlichen Fachpersonal bzw. Testcenter/Gesundheitsamt an, um Ihr Testergebnis zu erhalten. Wird Ihnen dabei ein positives Testergebnis mitgeteilt, können Sie über die TAN-Hotline eine teleTAN beantragen (<a href='#international_phone_numbers' target='_blank' >klicken Sie hier um zu den Telefonnummern zu gelangen</a>), um über die Corona-Warn-App Ihre Kontakte zu warnen. Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ??????h eingestellt. Bevor Sie die teleTAN in die App (Version 2.28 oder darunter) eintragen können, müssen Sie erst das noch ausstehende Testergebnis aus der App löschen. Dazu tippen Sie bitte auf ",
```

File: src/data/faq_de.json [hunk 5]
```diff
@@ -1789,9 +1811,12 @@
                                 "anchor": "hearing_impaired",
                                 "active": false,
                                 "textblock": [
+                                    "<b>Aktualisiert am ?????r</b>",
+                                    "Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ?????h eingestellt.",
```
Comment (larswmh) [inline src/data/faq_de.json:1815]: Specified placeholder was ??????h, one ? is missing
```suggestion
                                    "Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ??????h eingestellt.",
```

File: src/data/faq_de.json [hunk 6]
```diff
@@ -2804,17 +2829,19 @@
...
+                                    "Darüber hinaus gibt es weitere technische Maßnahmen, z.&nbsp;B. indem nach dem Zufallsprinzip unechte Meldungen erstellt und gesendet und dann im Backend wieder verworfen werden. Diese unechten Meldungen sind von den gültigen Meldungen nicht zu unterscheiden und erzeugen eine Art Hintergrundgeräusch, durch das der Sendeaufruf der TANs (Ab der Version 3.0 der Corona-Warn-App wird keine teleTAN mehr benötigt, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde)  und das Hochladen der Infektionsdaten nicht von diesem Hintergrundgeräusch unterscheidbar sind. So kann selbst bei überwachtem Netzwerkverkehr eine plausible Abstreitbarkeit geschaffen werden."
```
Comment (larswmh) [inline src/data/faq_de.json:2834]: Unnecessary space
```suggestion
                                    "Darüber hinaus gibt es weitere technische Maßnahmen, z.&nbsp;B. indem nach dem Zufallsprinzip unechte Meldungen erstellt und gesendet und dann im Backend wieder verworfen werden. Diese unechten Meldungen sind von den gültigen Meldungen nicht zu unterscheiden und erzeugen eine Art Hintergrundgeräusch, durch das der Sendeaufruf der TANs (Ab der Version 3.0 der Corona-Warn-App wird keine teleTAN mehr benötigt, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde) und das Hochladen der Infektionsdaten nicht von diesem Hintergrundgeräusch unterscheidbar sind. So kann selbst bei überwachtem Netzwerkverkehr eine plausible Abstreitbarkeit geschaffen werden."
```

File: src/data/faq_de.json [hunk 7]
```diff
@@ -3679,7 +3706,7 @@
                 {
                     "term": "TAN",
                     "anchor": "tan",
-                    "description": "10-stellige Buchstaben-Zahlenkombination zur Meldung positiver Testergebnisse. Diese sind über die TAN-Hotline anzufragen. <b>National:</b> 0800 7540002 und <b>international:</b> +49 30 498 75402"
+                    "description": "<p>Aktualisiert am ?????r.</p> 10-stellige Buchstaben-Zahlenkombination zur Meldung positiver Testergebnisse. Diese sind über die TAN-Hotline anzufragen. <b>National:</b> 0800 7540002 und <b>international:</b> +49 30 498 75402. <p>Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ?????h eingestellt.</p><a href='#warn_without_tan' target='_blank'>Wie warne ich ohne teleTAN?</a>"
```
Comment (larswmh) [inline src/data/faq_de.json:3709]: Specified placeholder was ??????h, one ? is missing
Update note should be in bold text and without a "." at the end of it, just like in FAQ entries
```suggestion
                    "description": "<p><b>Aktualisiert am ?????r</b></p> 10-stellige Buchstaben-Zahlenkombination zur Meldung positiver Testergebnisse. Diese sind über die TAN-Hotline anzufragen. <b>National:</b> 0800 7540002 und <b>international:</b> +49 30 498 75402. <p>Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ??????h eingestellt.</p><a href='#warn_without_tan' target='_blank'>Wie warne ich ohne teleTAN?</a>"
```

File: src/data/faq.json [hunk 8]
```diff
@@ -1789,9 +1817,13 @@
                                 "anchor": "hearing_impaired",
                                 "active": false,
                                 "textblock": [
+                                    "<b>Updated ?????r</b>",
+                                    "Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of ?????h.",
```
Comment (larswmh) [inline src/data/faq.json:1821]: Specified placeholder was ??????h, one ? is missing
```suggestion
                                    "Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of ??????h.",

```

File: src/data/faq.json [hunk 9]
```diff
@@ -1584,16 +1609,19 @@
...
+                                    "In this case, please call the responsible medical staff or test center/health authority to get your test result. If you are informed of a positive test result, you can request a teleTAN from the TAN Hotline (<a href='#international_phone_numbers' target='_blank' >click here to see our phone numbers</a>) to warn your contacts via the Corona-Warn-App. Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of ?????h. Before you are able to enter the teleTAN into the app (version 2.28 or smaller), you have to delete the pending test result from the app. To do this, please tap on",
```
Comment (larswmh) [inline src/data/faq.json:1618]: Specified placeholder was ??????h, one ? is missing
```suggestion
                                    "In this case, please call the responsible medical staff or test center/health authority to get your test result. If you are informed of a positive test result, you can request a teleTAN from the TAN Hotline (<a href='#international_phone_numbers' target='_blank' >click here to see our phone numbers</a>) to warn your contacts via the Corona-Warn-App. Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of ??????h. Before you are able to enter the teleTAN into the app (version 2.28 or smaller), you have to delete the pending test result from the app. To do this, please tap on",
```

File: src/data/faq.json [hunk 10]
```diff
@@ -1521,7 +1543,10 @@
                                     "<b>TAN Hotline</b>",
                                     "The TAN hotline will help you to enter the TAN if you have a positive PCR test result and if you have not received a TAN or document with a PCR-Test QR code. The TAN hotline is available Monday to Sunday from 7 a.m. to 8 p.m. German local time:",
                                     "<b>National:</b> 0800 7540002",
-                                    "<b>International:</b> +49 30 498 75402"
+                                    "<b>International:</b> +49 30 498 75402",
+                                    "Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of  ?????h.",
```
Comment (larswmh) [inline src/data/faq.json:1547]: Specified placeholder was ??????h, one ? is missing
```suggestion
                                    "Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of ??????h.",
```

**Review by larswmh — CHANGES_REQUESTED** [review]
In addition to my comments, the English TAN Glossary still needs to be updated

File: src/data/faq.json [hunk 11]
```diff
@@ -3659,7 +3693,7 @@
                 {
                     "term": "TAN",
                     "anchor": "tan",
-                    "description": "10-digit letter number combination for reporting positive test results. These can be requested via the TAN hotline. <b>National:</b> 0800 7540002 and <b>international:</b> +49 30 498 75402"
+                    "description": "<p>Updated ??????r.</p>10-digit letter number combination for reporting positive test results. These can be requested via the TAN hotline. <b>National:</b> 0800 7540002 and <b>international:</b> +49 30 498 75402. <p>Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of  ??????h.</p><a href='#warn_without_tan' target='_blank' >How do I warn without teleTAN?</a>"
```
Comment (larswmh) [inline src/data/faq.json:3696]: Remove one ? and punctuation from ??????r, make it bold
Remove an unnecessary space before ??????h
```suggestion
                    "description": "<p><b>Updated ?????r</b></p>10-digit letter number combination for reporting positive test results. These can be requested via the TAN hotline. <b>National:</b> 0800 7540002 and <b>international:</b> +49 30 498 75402. <p>Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of ??????h.</p><a href='#warn_without_tan' target='_blank' >How do I warn without teleTAN?</a>"
```

**Comment by MikeMcC399** [comment]
@larswmh 
Did you consider using gulp variables for the unknown dates?
That would make it easier to test instead of using `??????r` etc. as pseudo-variables.

As in:
https://github.com/corona-warn-app/cwa-website/blob/bf6a6655f838bdaefdaf8f8becd6577f1ce33bcf/gulpfile.mjs#L525-L531

**Comment by larswmh** [comment]
@MikeMcC399 

> Did you consider using gulp variables for the unknown dates?
> That would make it easier to test instead of using ??????r etc. as pseudo-variables.

I'm not sure if I understand this correctly. For testing and reviewing, I don't think it the variable matters. As seen in this PR, I could have come up with better ones, as they can easily be confused, but since we're now already in the process, we should stick to them.
Our plan is to merge all of these PRs with these variables into the release/3.0 branch before the release, and then replace all those variables with the correct date at once right before merging into master

File: src/data/faq.json [hunk 12]
```diff
@@ -71,8 +71,9 @@
...
...certain requirements that must be met:<ul><li>After receiving the positive test result, it must be voluntarily shared by the user via the Corona-Warn-App. If the test result could not be determined via the app, the positive PCR test result can be entered and shared via a TAN. Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App.</li><li>The app must have been installed and active on both devices <b>before</b> sharing the positive test result via the Corona-Warn-App, otherwise no encounters can have been recorded.</li><li>The two devices must have been close enough and long enough <b>before</b> sharing the test result. Encounters <b>after</b> sharing a positive result are of course <b>not</b> warned.</li><li>The time period for the onset of the first symptoms given when sharing the test result, as well as the type or intensity of the symptoms, may also play a role in the risk calculation. So it also depends on which days before sharing the test result the person's devices recorded the encounters. In some cases, this may result in no warning being sent.</li></ul>",
```
Comment (MikeMcC399) [inline src/data/faq.json:75]: In
"Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App."

"teleTAN" = "a teleTAN"

Comment (MikeMcC399) [inline src/data/faq.json:75]: In

"The two devices must have been close enough and long enough <b>before</b> sharing the test result."

"close enough and long enough" =>
"close enough and for long enough"

File: src/data/faq.json [hunk 13]
```diff
@@ -158,9 +159,11 @@
...
...mbers, for example, for your children who do not have their own phones, you can use them on your own device as well since version 2.21. There is no need to wait between scanning your own test results and those of family members. They are processed independently and their status can be viewed on the Status Tab.</li><li>If your QR code doesn't work and you received a positive test result: Call the TAN hotline (<a href='#international_phone_numbers' target='_blank' >click here to see the phone numbers</a>) to receive a teleTAN. You can then enter the teleTAN in your Corona-Warn-App so that the persons you encountered in the past days can be informed over the Corona-Warn-App. Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of ??????h. A warning can only be issued for a test result that is assigned to the owner of the device. You cannot warn on behalf of family members whose tests you have added to your device. Please pay attention to the recommendations the Corona-Warn-App displays.</li></ol>",
```
Comment (MikeMcC399) [inline src/data/faq.json:164]: In

"Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App."

"teleTAN" => "a teleTAN"

File: src/data/faq.json [hunk 14]
```diff
@@ -274,7 +277,10 @@
...
                                 "active": false,
                                 "textblock": [
-                                    "It is not possible to share a positive <a href='#glossary_rapid_antigen_test' target='_blank'>rapid antigen test</a>, <a href='#glossary_selftest' target='_blank'>self-test</a> or <a href='#glossary_rapid_pcr_test' target='_blank'>rapid PCR test</a> result via the TAN hotline."
+                                    "<b>Updated ?????r</b>",
+                                    "Starting with version 3.0 of the Corona-Warn-App, a teleTAN is no longer required to share a positive test result that has not been submitted to the Corona-Warn-App. That is, you can share your rapid antigen test result or rapid PCR test result (point-of-care PCR tests) without a TAN through the app.",
+                                    "With the versions 2.28 and before it is not possible to get a positive result from a<a href='#glossary_rapid_antigen_test' target='_blank'>rapid antigen test</a>, <a href='#glossary_selftest' target='_blank'>self-test</a> or <a href='#glossary_rapid_pcr_test' target='_blank'>rapid PCR test</a> result via the TAN hotline.",
```
Comment (MikeMcC399) [inline src/data/faq.json:282]: `from a<a` missing space before `<`

File: src/data/faq.json [hunk 15]
```diff
@@ -335,18 +341,26 @@
...
                                     "If you have more than one smartphone (e.g. work and private), we recommend that you install the CWA on your private device first. Also, when scanning PCR-Test QR codes and sharing a positive PCR test result, we recommend that you first use your private device. If you additionally want to share a positive PCR test result via your company device or any other additional smartphone, this can be done via a so-called teleTAN. By contacting the TAN hotline 0800 7540002 with a positive PCR test result, you will receive a 10-digit TAN with which the PCR test result can also be registered in the app. <b>Please note:</b> The TAN number is only valid for one hour.",
-                                    "See also <a href='#international_phone_numbers' target='_blank' >How to reach us from abroad</a>."
+                                    "Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of ??????h.",
```
Comment (MikeMcC399) [inline src/data/faq.json:347]: "teleTAN" => "a teleTAN"

File: src/data/faq.json [hunk 16]
```diff
@@ -335,18 +341,26 @@
...
                                 "active": false,
                                 "textblock": [
+                                    "<b>Updated ?????r</b>",
                                     "If you are in another EU country or in one of <a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Tests.html' target='_blank' rel='noopener noreferrer'>these countries</a> and have made a molecular biological test (PCR test) which is positive, you can upload it to the Corona-Warn-App. To do so, it is necessary to verify your identity by calling the TAN hotline 0800 7540002. You will receive a 10-digit TAN with which the test result can be registered in the app. Attention: The TAN is only valid for one hour!",
-                                    "See also <a href='#international_phone_numbers' target='_blank' >How to reach us from abroad</a>."
+                                    "Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of ??????h.",
```
Comment (MikeMcC399) [inline src/data/faq.json:360]: "teleTAN" => "a teleTAN"

**Comment by MikeMcC399** [comment]
In the English text there are many places which use "teleTAN" without the indefinite article. It should be "a teleTAN". I have only marked up some of them. They should however all be changed.

File: src/data/faq.json [hunk 17]
```diff
@@ -1584,16 +1608,19 @@
...
+                                    "In this case, please call the responsible medical staff or test center/health authority to get your test result. If you are informed of a positive test result, you can request a teleTAN from the TAN Hotline (<a href='#international_phone_numbers' target='_blank' >click here to see our phone numbers</a>) to warn your contacts via the Corona-Warn-App. Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of ??????h. Before you are able to enter the teleTAN into the app (version 2.28 or smaller), you have to delete the pending test result from the app. To do this, please tap on",
```
Comment (MikeMcC399) [inline src/data/faq.json:1617]: not "smaller"
=> "earlier" or "lower"

File: src/data/faq.json [hunk 18]
```diff
@@ -2795,17 +2827,19 @@
...
-                                    "Furthermore, various additional technical measures have been taken. For example, false notifications that will be discarded on the server side are randomly created and sent. These false notifications cannot be distinguished from valid notifications and create a type of background noise. The transmission of TANs and upload of real infection data cannot be distinguished from this background noise. So, even with monitored network traffic, plausible deniability is still possible."
+                                    "Furthermore, various additional technical measures have been taken. For example, false notifications that will be discarded on the server side are randomly created and sent. These false notifications cannot be distinguished from valid notifications and create a type of background noise. The transmission of TANs (As of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been submitted to the Corona-Warn-App) and upload of real infection data cannot be distinguished from this background noise. So, even with monitored network traffic, plausible deniability is still possible."
```
Comment (MikeMcC399) [inline src/data/faq.json:2832]: "(As of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been submitted to the Corona-Warn-App)"

This sentence is in the middle of another sentence. You could leave it there if you change "As" to "as", however that is quite untidy.

I suggest to move it to the end of the paragraph and write:
"Note: As of version 3.0 of the Corona-Warn-App, a teleTAN is no longer used to share a positive test result that has not been submitted to the Corona-Warn-App."

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
See also in-line comments

File: src/data/faq.json (hunk 18 above)
Comment (dsarkar) [reply src/data/faq.json]: Thanks for suggestion, has been adopted.

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for reviewing !

**Review by dsarkar — APPROVED** [review]

**Pull request merged**